### DISCUSSION TITLE
After Docker Installation, No Other Folders

### DISCUSSION THREAD

**Discussion by Buuurak** [AzuraCast/AzuraCast#7270] [General]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

 v0.20.1

### What happened?

After the Docker Installation, are there at /var/azuracast/ no other folders like stations/radio/media
![image](https://github.com/AzuraCast/AzuraCast/assets/98275029/a4b9ba39-c357-4e92-a96d-aba290a38d4f)

**Comment by gAlleb** [comment]
1. https://docs.docker.com/guides/docker-concepts/the-basics/what-is-a-container/
2. https://www.azuracast.com/docs/administration/docker/#mounting-a-directory-into-a-station -> `Storing Your Station Data on the Host Machine`

**Reply by LordHelmchen666** [reply]
you can find docker containers normally under /var/lib/docker/volumes ....
